components/fs-data-provider/tests/test_file_io.rs

```diff
@@ -0,0 +1,29 @@
+use icu_data_provider::prelude::*;
+use icu_data_provider::structs;
+use icu_fs_data_provider::FsDataProvider;
+use std::borrow::Cow;
+
+#[test]
+fn test_read_json() {
+    let provider = FsDataProvider::try_new("tests/testdata/json_plurals_37").unwrap();
+    let response = provider
+        .load(&DataRequest {
+            data_key: icu_data_key!(plurals: cardinal@1),
+            data_entry: DataEntry {
+                variant: None,
+                langid: "sr".parse().unwrap(),
+            },
+        })
+        .unwrap();
+    let plurals_data: &structs::plurals::PluralRuleStringsV1 = response.borrow_payload().unwrap();
```

Comment:
can you use `expect`? Our style guide recommends migrating all `unwrap` to `expect` before landing :)